asapdiscovery-data/asapdiscovery/data/scripts/download_fragalysis_data.py

```diff
@@ -17,7 +24,11 @@ def get_args():
 def main():
     args = get_args()
 
-    download(args.o, args.x)
+    # Copy the base call and update the base target with the cli-specified target
+    api_call = copy.deepcopy(API_CALL_BASE)
+    api_call["target_name"] = args.target.capitalize()
+
+    download(args.o, API_CALL_BASE, args.x)
```

Comment:
```suggestion
    download(args.o, api_call, args.x)
```

Reply:
Yes, totally! This is what happens when you try to make changes in the middle of a talk in a conference :)